app/components/e-radio.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+  <div :class="b(modifiers)">
+    <label :class="b('label')"
+           @mouseenter="isHover = true"
+           @mouseleave="isHover = false">
+      <input :class="b('field')"
+             v-bind="$attrs"
+             :disabled="disabled"
+             :value="value"
+             v-model="internalValue"
+             :name="name"
+             type="radio"
+             @change="onChange"
+      >
+      <label :class="b('fake-button')"></label>
```

Comment:
- [ ] Do you need here a `<label>`? It's not wrapping or bind via "for" to an element. I would recommend to us a `<span>` here